presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -225,6 +228,20 @@ public SystemSessionProperties(
                         COLOCATED_JOIN,
                         "Experimental: Use a colocated join when possible",
                         featuresConfig.isColocatedJoinsEnabled(),
+                        false),
+                new PropertyMetadata<>(
+                        OPERATOR_MEMORY_LIMIT_BEFORE_SPILL,
+                        "Experimental: Operator memory limit before spill",
+                        VARCHAR,
+                        DataSize.class,
+                        featuresConfig.getOperatorMemoryLimitBeforeSpill(),
+                        false,
+                        value -> DataSize.valueOf((String) value),
+                        DataSize::toString),
+                longSessionProperty(
+                        MAX_ENTRIES_BEFORE_SPILL,
+                        "Experimental: Limit max number of entries before spill",
```

Comment:
@cberner hmm, maybe I forgot about something. This property is for testing purposes so it can be more reliable and stable compared to data size. It's difficult to guess desired size in bytes before spill for tests. Would single entry fit in ~100 bytes limit? How many entries fit in 1000 bytes? 1? 10? 50? Since some tests  performs aggregations with only couple of unique values it's critical to pick value small enough to force spill for every query. 

Alternative solution would be to drop `MAX_ENTRIES_BEFORE_SPILL` and for tests use 1 byte memory limit, but this limits us the number of test cases that we can cover.